prow/plugins/open/open.go

```diff
@@ -0,0 +1,71 @@
+/*
+Copyright 2016 The Kubernetes Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package open
+
+import (
+	"regexp"
+
+	"github.com/Sirupsen/logrus"
+
+	"k8s.io/test-infra/prow/github"
+	"k8s.io/test-infra/prow/plugins"
+)
+
+const pluginName = "open"
+
+var openRe = regexp.MustCompile(`(?mi)^\/open\r?$`)
+
+func init() {
+	plugins.RegisterIssueCommentHandler(pluginName, handleIssueComment)
+}
+
+type githubClient interface {
+	CreateComment(owner, repo string, number int, comment string) error
+	OpenIssue(owner, repo string, number int) error
+}
+
+func handleIssueComment(pc plugins.PluginClient, ic github.IssueCommentEvent) error {
+	return handle(pc.GitHubClient, pc.Logger, ic)
+}
+
+func handle(gc githubClient, log *logrus.Entry, ic github.IssueCommentEvent) error {
+	// Only consider closed issues and new comments.
+	if ic.Issue.State != "closed" || ic.Issue.IsPullRequest() || ic.Action != "created" {
+		return nil
+	}
+
+	if !openRe.MatchString(ic.Comment.Body) {
+		return nil
+	}
+
+	org := ic.Repo.Owner.Login
+	repo := ic.Repo.Name
+	number := ic.Issue.Number
+
+	commentAuthor := ic.Comment.User.Login
+
+	// Allow assignees and authors to re-open issues.
+	log.Infof("Comment author: %v", commentAuthor)
```

Comment:
Remove this line. Note that [the logger that is passed to the plugin](https://github.com/kubernetes/test-infra/blob/9e67b4cc40d2a70434300ecbd71c07b9b2e7bb72/prow/cmd/hook/events.go#L82-L88) already has this field set.